Makefile

```diff
@@ -31,7 +31,7 @@ clean:
 	rm -rf dist/*
 
 test:
-	go test -cover -v `glide novendor`
+	sh go.test.sh
```

Comment:
With this script, it becames unnecessary to rewrite scripts related to tests when creating packages other than `poll`, .
But now there is no plan creating other packages and no problem without it.

I'll remove `go.test.sh`.